infrastructures/openshift/src/main/java/org/eclipse/che/workspace/infrastructure/openshift/provision/OpenShiftServersConverter.java

```diff
@@ -36,6 +38,13 @@
 @Singleton
 public class OpenShiftServersConverter implements ConfigurationProvisioner<OpenShiftEnvironment> {
 
+  ExternalServerExposerStrategy openshiftExternalServerExposer;
```

Comment:
Please make this field private and generify it with OpenShiftEnvironment.

Reply:
class removed.